### ISSUE TITLE
Instagram sharing opens layout tool

### ISSUE THREAD

**Issue by sakerrr** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#627]
Hello, I have a problem with sharing via instagram. The plugin seems to open instagrams "layout" tool instead of the default first window where you can choose images from the gallery.

I'm just calling shareViaInstagram(url, .. 

Am I doing something wrong ?

**Comment by tavosansal** [comment]
Hello...I have this issue as well. Do you know of what could be happening or if you found a fix?

**Comment by sakerrr** [comment]
@tavosansal yes, it only happens when you have Layouts app installed, so you might call it a feature lol.

**Comment by tavosansal** [comment]
The weird thing is that I have tried to replicate it (with the Layout app installed) and I cannot get it to replicate. It always opens in Instagram for me. However I have users that do claim this happens to them. So I am not sure how to even begin debugging.
